core/utxo/utxo_meta.go

```diff
@@ -166,6 +166,28 @@ func (uv *UtxoVM) GetForbiddenContract() *pb.InvokeRequest {
 	return uv.meta.GetForbiddenContract()
 }
 
+func (uv *UtxoVM) GetGroupChainContract() *pb.InvokeRequest {
```

Comment:
I don't suggest to provide functions for different data in UTXO, it's too heavy. I think we can expose a get/set interface in UTXO and passing the key/value of the data to make UTXO more clear.

Reply:
It is same with ForbiddenContract.